options/options.js

```diff
@@ -1,9 +1,16 @@
+"use strict";
+// Defaults
+var DecoderMemoryLimitInMegabytes = 32;
+var DebugOutput = false;
```

Comment:
This should probably be `AllowDebugOutput` or `DebugOutputEnabled` or some such to better indicate that it is the option value and not, for example, a string of current debug output :)